src/averageAges.js

```diff
@@ -55,7 +82,32 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const arrOfDiff = [];
```

Comment:
Replace the `arrOfDiff` array with a reduce function to calculate the sum of age differences in a more efficient way.